build-test.xml

```diff
@@ -16791,6 +16791,8 @@ CATALINA_OPTS="$CATALINA_OPTS -Dcom.sun.management.jmxremote -Dcom.sun.managemen
 			</then>
 		</if>
 
+		<delete dir="${app.server.tomcat.dir}/work" includeemptydirs="true" />
```

Comment:
Hi @shuyangzhou , this is to make sure license test can start up the portal while running the ci test